sdk/circomlib/tests/jwt.test.ts

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+import fs from 'fs'
+import path from 'path'
+import { pki } from 'node-forge'
+import { wasm as wasm_tester } from 'circom_tester'
+import { padString, toCircomBigIntBytes } from '../src'
+
+describe('JWT Test', () => {
```

Comment:
Will the CI run these tests automatically?

Reply:
I'll add it later